$NetBSD: patch-Makefile,v 1.1 2019/02/28 19:27:43 leot Exp $

- Add support for MANDIR instead of forcing install of man pages in ${PREFIX}/man.
- Use INSTALL_MAN instead of INSTALL to install man pages with the proper
  permissions (no executable bit set)

--- Makefile.orig	2019-02-27 19:07:35.000000000 +0000
+++ Makefile
@@ -68,6 +68,7 @@ DISTFILES+=	tests/pick-test.c
 DISTFILES+=	tests/pick-test.sh
 
 PREFIX=	/usr/local
+MANDIR=	man
 
 all: ${PROG}
 
@@ -100,8 +101,8 @@ distclean: clean
 install: ${PROG}
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	${INSTALL} ${PROG} ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/man/man1
-	${INSTALL} ${.CURDIR}/pick.1 ${DESTDIR}${PREFIX}/man/man1
+	@mkdir -p ${DESTDIR}${PREFIX}/${MANDIR}/man1
+	${INSTALL_MAN} ${.CURDIR}/pick.1 ${DESTDIR}${PREFIX}/${MANDIR}/man1
 .PHONY: install
 
 test: ${PROG}
